src/components/vm/VM.tsx

```diff
@@ -0,0 +1,102 @@
+import { isValidAttribute } from 'dompurify';
+import { mapValues } from 'lodash';
+import { EthersProviderContext, useAccount, useInitNear, utils, Widget } from 'near-social-vm';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import React, { useContext, useEffect, useState } from 'react';
+
+import { useEthersProviderContext } from '@/data/web3';
+import { useAuthStore } from '@/stores/auth';
+import { cookiePreferences, optOut, recordHandledError, recordWalletConnect } from '@/utils/analytics';
+import { commitModalBypassAuthorIds, commitModalBypassSources, isLocalEnvironment, networkId } from '@/utils/config';
+
+import { NearContext } from '../WalletSelector';
+
+export default function Component(props: any) {
```

Comment:
Let's stay away from default exports and favor named exports: https://dev.to/phuocng/avoid-using-default-exports-a1c

Reply:
I tried changing it to a export instead of a default, but then the `dynamic(() => import('./VM'), { ... }` fails

Reply:
Oh sure, dynamic imports are a bit weird. Feel free to ignore this :)